packages/swapper/src/swappers/JupiterSwapper/swapperApi/getTradeRate.ts

```diff
@@ -114,6 +137,52 @@ export const getTradeRate = async (
     return { networkFeeCryptoBaseUnit: fast.txFee }
   }
 
+  const buyAssetAddress =
+    buyAsset.assetId === solAssetId
+      ? fromAssetId(wrappedSolAssetId).assetReference
+      : fromAssetId(buyAsset.assetId).assetReference
+
+  const sellAssetAddress =
+    sellAsset.assetId === solAssetId
+      ? fromAssetId(wrappedSolAssetId).assetReference
+      : fromAssetId(sellAsset.assetId).assetReference
+
+  const [buyAssetReferralPubKey] = await PublicKey.findProgramAddressSync(
+    [
+      Buffer.from('referral_ata'),
+      new PublicKey(SHAPESHIFT_JUPITER_REFERRAL_KEY).toBuffer(),
+      new PublicKey(buyAssetAddress).toBuffer(),
+    ],
+    new PublicKey(JUPITER_AFFILIATE_CONTRACT_ADDRESS),
+  )
+
+  const [sellAssetReferralPubKey] = await PublicKey.findProgramAddressSync(
```

Comment:
```suggestion
  const [sellAssetReferralPubKey] = PublicKey.findProgramAddressSync(
```

Reply:
Done in [10c98b6](https://github.com/shapeshift/web/pull/8267/commits/10c98b6156314e71f8ec309e25bbd54fd475ec53)